draft-ietf-dnsop-dns-capture-format-01.md

```diff
@@ -845,6 +856,19 @@ The timestamp of a list item is that of the query for cases 1 and 2 and that of
 
 When ending capture, all remaining entries in the Q/R data item FIFO should be treated as timed out queries.
 
+## Malformed Packet Considerations
+
+As noted in (#malformed-packets), partially malformed packets go through query and response matching. They have
+a Primary ID, and possibly a Secondary ID, and can be matched in the same way as well-formed packets.
+
+When a partially malformed packet is matched, the Q/R data item generated by this process must be populated with
+the information in the Primary ID, the Secondary ID (if relevant) and the packet timestamp, and a flag
+set to indicate that the packet was malformed and the Q/R data item may therefore be incomplete. All other fields in the
+Q/R data item are optional, and so may be omitted. Implementations may choose to include the contents of other decoded
```

Comment:
Shall we just move this back into the Malformed section just after the Rationals?